pkg/interactor/interactor.go

```diff
@@ -19,11 +19,11 @@ type Interactor interface {
 // DataInteractor represents the data layer interactor structure
 type DataInteractor struct {
 	things    things.Lister
-	DataStore *data.Store
+	DataStore data.IDataStore
```

Comment:
I think you can group these changes with the commit _a9dd79ba0c281189bb54a3bdfd981ca1d3f4cd97_ ones.

Reply:
Done!